### DISCUSSION TITLE
kata level

### DISCUSSION THREAD

**Discussion by alihgad** [codewars/codewars.com#3112] [General]
when i be better to change kata hard level

**Comment by d3n7** [comment]
You can do any level kata you want to.

**Comment by SyNIXD06** [comment]
In my opinion, I consider starting from the bottom when you feel free to do any kata, try one from a higher level and see how it goes. ツツ

**Comment by ahmadshah2103** [comment]
You can solve any level kata but you should diffinitely start from the level you are confident at.
HIgher _kyu_ kata means its easy.
The only restriction is that you cannot see the solution of katas that are above your level (lower _kyu_)

**Comment by Pilicirce** [comment]
Hi everyone! I am new here (in Codewars) and I would like to ask you something regarding the levels: 
I started as level 8kyu and now I am doing exercises from level 7kyu. But this is not updated in my page. why? is it automatically? should I do anything? I would like that level 7kyu appears in my budge
Thanks

**Reply by NoRoSo** [reply]
Sorry for the late reply!

Your badge only ever updates once you reach a certain threshold on your points. Once you get a certain number of points (or honor), it will update.

**Comment by MurariAmbati** [comment]
Do any level in Kata
